src/components/StackNameGenerator.jsx

```diff
@@ -22,10 +22,11 @@ class StackNameGenerator extends React.Component {
       url: this.URL + letter,
       responseType: "text"
     }).then(async response => {
-      let results = response.data.split("\n");
-      let filteredResults = results.filter(word => word != null && word !== "");
-      const randomIndex = Math.floor(Math.random() * filteredResults.length);
-      return filteredResults[randomIndex];
+      const results = response.data.split("\n");
+      const filteredResults = results.filter(word => word != null && word !== "");
+      const deDupeResults = filteredResults.filter(word => !this.state.generatedList.includes(word));
```

Comment:
reduces list of values to ones not yet used; if size is greater than 0, uses it for base for randomizer; if size is 0, simply uses the full list

Reply:
This doesn't totally work because at this state in the runtime, the `this.state.generatedList` != `listOfThings` because `this.state.generatedList` refers to that last word that was produced, not the current one being built. You might be able to see that in this build: 
https://deploy-preview-5--condescending-heisenberg-6c385d.netlify.com/

Reply:
While that is true that the words are not the same, it doesn't truly cause any problems, and, may even result in a desired experience. Consider this, "Bike" results in one stack, changing the B to an M results in the same output, except that the first item is swapped.

I suppose it really comes down to what the desired result is.

I'll also take a look at the other suggestion at my earliest opportunity.